pkg/renamepvc/renamepvc.go

```diff
@@ -56,22 +60,32 @@ Afterwards the old PVC is automatically deleted.`,
 		Args:         cobra.ExactArgs(2), //nolint: gomnd // needs always 2 inputs
 		SilenceUsage: true,
 		RunE: func(c *cobra.Command, args []string) error {
-			return o.run(c.Context(), args[0], args[1])
+			var err error
+			o.sourceNamespace, _, err = o.configFlags.ToRawKubeConfigLoader().Namespace()
+			if err != nil {
+				return err
+			}
+
+			if o.targetNamespace == "" {
+				o.targetNamespace = o.sourceNamespace
+			}
+
+			o.oldName = args[0]
+			o.newName = args[1]
+
+			return o.run(c.Context())
 		},
 	}
 	cmd.Flags().BoolVarP(&o.confirm, "yes", "y", false, "Skips confirmation if flag is set")
+	cmd.Flags().StringVar(&o.targetNamespace, "target-namespace", "",
```

Comment:
Maybe it would be nice to include the shorthand "-N" to be consistent with pv-migrate.